.concourse/pipeline.yaml.gomplate

```diff
@@ -353,11 +352,7 @@ jobs:
         - -ce
         - |
           cd kubecf-{{ $branch }}
-          ./dev/linters/shellcheck.sh
-          ./dev/linters/yamllint.sh
-          ./dev/linters/helmlint.sh
-          bazel test //rules/kubecf:create_sample_values_test
-          bazel test //deploy/helm/kubecf:values_doc_test
+          make lint
```

Comment:
Does `make -C kubecf-{{ $branch }} lint` work? If yes, we won't need the `/bin/bash` anymore, I think?

Reply:
True, changed. Leaving the `/bin/bash` for build step, making a one-liner there muddies things.